CHANGES/9394.packaging.rst

```diff
@@ -0,0 +1,3 @@
+Switched to using the ``propcache`` package for property caching -- by :user:`bdraco`.
+
+The ``propcache`` package is derived from the property caching code in ``yarl`` and has been broken out to avoid maintaining it for multiple projects.
```

Comment:
```suggestion
Switched to using the :mod:`propcache` package for property caching -- by :user:`bdraco`.

The :mod:`propcache` package is derived from the property caching code in :mod:`yarl`
and has been broken out to avoid maintaining it for multiple projects.
```

Reply:
I think we need to do `:mod:`propcache.api`` now

Reply:
That might be true. Let me check...

Reply:
```console
$ python -Im sphinx.ext.intersphinx https://propcache.aio-libs.org/en/latest/objects.inv       
py:function
    propcache.api.cached_property                                                    : api/#propcache.api.cached_property
    propcache.api.under_cached_property                                              : api/#propcache.api.under_cached_property
py:module
    propcache.api                                                                    : api/#module-propcache.api
std:doc
    api                                      Reference                               : api/
    changes                                  Changelog                               : changes/
    contributing/guidelines                  Contributing docs                       : contributing/guidelines/
    contributing/release_guide               Release Guide                           : contributing/release_guide/
    index                                    propcache                               : 
std:label
    adding change notes with your prs        Adding change notes with your PRs       : contributing/guidelines/#adding-change-notes-with-your-prs
    genindex                                 Index                                   : genindex/
    modindex                                 Module Index                            : py-modindex/
    propcache-api                            Reference                               : api/#propcache-api
    propcache_changes                        Changelog                               : changes/#propcache-changes
    py-modindex                              Python Module Index                     : py-modindex/
    search                                   Search Page                             : search/
```

Reply:
Alright, I updated the refs in the suggestion.

Reply:
Thats much nicer, thanks.  I'll come back to this if I'm not too tired after I wrap up downstream effects of v1.0.0 propcache